src/widgets/activity-group/model/hooks/mocks/mocksForEntities.ts

```diff
@@ -16,6 +16,7 @@ export const progress: Progress = {
         startAt: new Date(2023, 1, 12, 14, 16, 17),
         endAt: new Date(2023, 1, 12, 15, 26, 17),
         availableTo: null,
+        entityName: 'mock-entity-name',
```

Comment:
Change it to empty string, please. The related BL doesn't use this field. And if even so, it'd be easy to catch.